doc/userguide/rules/payload-keywords.rst

```diff
@@ -274,6 +274,28 @@ You can also use the negation (!) before isdataat.
 
 .. image:: payload-keywords/isdataat1.png
 
+absent
+------
+
+The keyword ``absent`` checks that a sticky buffer does not exist.
+It can take an argument "only" to match only on absent buffer :
```

Comment:
What about `http.referer; absent; sid:1; rev:1;`?

Reply:
Tried that, but apparently a signature keyword must have either an argument or none, cannot be optional...